docs/basic-features/built-in-css-support.md

```diff
@@ -15,7 +15,7 @@ For example, consider the following stylesheet named `styles.css`:
 
 ```css
 body {
-  font-family: 'SF Pro Text', 'SF Pro Icons', system-ui;
+  font-family: sans-serif;
```

Comment:
Hmm, this was partially intentional to show the property would get compiled.

Reply:
@Timer If we want to show that the style will change, how about `monospace`? 

Or [we can use the stack we use on Next.js site](https://github.com/chibicode/next-site/blob/a75c9d189746d108433e69bcedaec2cd734f0125/components/page.js#L41-L42) (which doesn't use `system-ui`).